static/ensure_deps.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+CAIRO_VERSION=1.12.18
+FREETYPE_VERSION=2.6
+GIFLIB_VERSION=4.2.3
+LIBJPEG_VERSION=1.2.1
+LIBPNG_VERSION=1.2.56
+PIXMAN_VERSION=0.32.6
+ZLIB_VERSION=1.2.8
+
+CAIRO_URL=http://cairographics.org/releases/cairo-$CAIRO_VERSION.tar.xz
+FONTCONFIG_URL=http://cgit.freedesktop.org/fontconfig/plain/fontconfig/fontconfig.h
+FREETYPE_URL=http://download.savannah.gnu.org/releases/freetype/freetype-$FREETYPE_VERSION.tar.gz
+GIFLIB_URL=http://sourceforge.net/projects/giflib/files/giflib-4.x/giflib-$GIFLIB_VERSION.tar.gz
+LIBJPEG_URL=http://downloads.sourceforge.net/project/libjpeg-turbo/$LIBJPEG_VERSION/libjpeg-turbo-$LIBJPEG_VERSION.tar.gz
+LIBPNG_URL=http://sourceforge.net/projects/libpng/files/libpng12/$LIBPNG_VERSION/libpng-$LIBPNG_VERSION.tar.xz
+PIXMAN_URL=http://cairographics.org/releases/pixman-$PIXMAN_VERSION.tar.gz
+ZLIB_URL=http://zlib.net/zlib-$ZLIB_VERSION.tar.gz
```

Comment:
I'm just working on that, I'll push a commit with the fixes when it's ready, sorry for the inconvenience.